admin_interface/locale/es/LC_MESSAGES/django.po

```diff
@@ -19,172 +19,170 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: admin_interface/admin.py:18
-msgid "Enviroment"
-msgstr "Entorno"
+#: admin.py:18
+#, fuzzy
+#| msgid "Enviroment"
```

Comment:
You can delete the lines with fuzzy and the old msgid.

Reply:
Done!